packages/api/src/api/project/id/index.ts

```diff
@@ -0,0 +1,6 @@
+import { Router } from 'express';
+import { get } from './get';
+
+export const details = Router({ mergeParams: true });
```

Comment:
```suggestion
export const id = Router({ mergeParams: true });
```

Reply:
i was thinking the same but i just felt details would be better, but i see the idea is to keep the directory name and router name the same just as Charlie also told me.